composer.json

```diff
@@ -17,7 +17,10 @@
         "phpunit/phpunit": "4.7.*"
     },
     "autoload": {
-        "psr-0": { "PhpOption\\": "src/" }
+        "psr-0": {
+            "PhpOption\\": "src/",
+            "PhpOption\\Tests\\": "tests/"
```

Comment:
Please revert this, and instead add this to the autoloader in the bootstrap.php file for tests:

```php
$loader = require $autoloadFile;
$loader->add('PhpOption\\Tests\\', __DIR__);
```